neurom/io/hdf5.py

```diff
@@ -184,29 +184,29 @@ def remove_duplicate_points(points, groups):
         '''
 
         group_initial_ids = groups[:, 0]
+        group_len = len(group_initial_ids)
 
         def _find_last_point(group_id):
             ''' Identifies and returns the id of the last point of a group'''
+            return group_initial_ids[group_id + 1] - 1
```

Comment:
@juanchopanza Thanks for confirming this issue. I'm making the same assumption, so it's good to verify this!
@mgeplf If what Juan states is correct, I think we don't require more ordering assumptions. In Nascii format this ordering is important because we don't have ids of parents.